packages/pkgpanda-api/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+source /opt/mesosphere/environment.export
+
+export WORK_DIR=/run/dcos/pkgpanda-api/
+export STOP_TIMEOUT=30  # seconds
+
+pip install --no-deps --install-option="--prefix=$PKG_PATH" --root=/ /pkg/src/gunicorn
+
+mkdir -p $PKG_PATH/{etc,bin}
+envsubst '$WORK_DIR' < /pkg/extra/pkgpanda-api.conf.py > $PKG_PATH/etc/pkgpanda-api.conf.py
```

Comment:
Can we just use a HEREDOC for this one?

Practically it feels like we should just be able to drop this entirely / have it be the default config value.